ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5454,6 +5454,47 @@ RDC:
   implements:
   - MONITORING
 
+CHWZOUC:
+  description: "Chilled water valve control with occupied/unoccupied setpoints and temperature monitoring."
+  is_abstract: true
+  opt_uses:
+  - chilled_supply_water_temperature_sensor
+  uses:  
+  - chilled_water_valve_percentage_command
+  - zone_air_temperature_sensor
+  - zone_air_occupied_temperature_setpoint
```

Comment:
Kind of odd seeing occupied temp setpoint and unoccupied cooling setpoint. shouldnt they both be cooling setpoints?